pkgs/applications/networking/cluster/argocd-autopilot/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "argocd-autopilot";
+  version = "0.2.25";
+  commit = "b769e2e9886c1355aedffee42077245d120ff4f1";
+
+  src = fetchFromGitHub {
+    owner = "argoproj-labs";
+    repo = "argocd-autopilot";
+    rev = "v${version}";
+    sha256 = "sha256-8KW+KgepKQKJQ7CU0Y2bhQuxlHFkOG6ow1g0d1Eu9TA=";
+  };
+
+  vendorSha256 = "sha256-SFmv4Cn3+toNiioMOxc6xIdDR3rzm3L3TM0jbuTGysw=";
```

Comment:
Please update to the latest (https://github.com/argoproj-labs/argocd-autopilot/releases/tag/v0.2.30).